kupfer/plugin/chromium.py

```diff
@@ -1,6 +1,6 @@
-__kupfer_name__ = _("Chromium Bookmarks")
+__kupfer_name__ = ("Chromium Bookmarks")
```

Comment:
_() translate string to user locale; can't be removed.

Reply:
Sorry, I didn't know that. Thanks.